src/connectdlg.cpp

```diff
@@ -887,3 +895,25 @@ void CConnectDlg::DeleteAllListViewItemChilds ( QTreeWidgetItem* pItem )
         delete pCurChildItem;
     }
 }
+
+void CConnectDlg::UpdateDirectoryServerComboBox()
+{
+    // directory server address type combo box
+    cbxCentServAddrType->clear();
+    cbxCentServAddrType->addItem ( csCentServAddrTypeToString ( AT_DEFAULT ) );
+    cbxCentServAddrType->addItem ( csCentServAddrTypeToString ( AT_ANY_GENRE2 ) );
+    cbxCentServAddrType->addItem ( csCentServAddrTypeToString ( AT_ANY_GENRE3 ) );
+    cbxCentServAddrType->addItem ( csCentServAddrTypeToString ( AT_GENRE_ROCK ) );
+    cbxCentServAddrType->addItem ( csCentServAddrTypeToString ( AT_GENRE_JAZZ ) );
+    cbxCentServAddrType->addItem ( csCentServAddrTypeToString ( AT_GENRE_CLASSICAL_FOLK ) );
+    cbxCentServAddrType->addItem ( csCentServAddrTypeToString ( AT_GENRE_CHORAL ) );
+
+    for ( int iLEIdx = 0; iLEIdx < MAX_NUM_SERVER_ADDR_ITEMS; iLEIdx++ )
```

Comment:
What's "LE"?  If it's "list entry" that doesn't add anything other than confusion.  Otherwise, I'd rather a longer name.

Reply:
I see where you got the "LE" from now...  I still don't like it...

Reply:
It's a verbatim cut-and-paste from the settings dialog.  I don't know what it stands for.

Reply:
iCustomDirectoryVectorIterator ?

Reply:
:) `i` would do, in my view.  There are places with `iIdx` (an integer index...) but I don't even like that...

Reply:
>  I'd rather a longer name.

yeah it's either i, or iCustomDirectoryVectorIteratorWithNameSuggestedByPljones